### PR TITLE
add tests for awardachievement API

### PR DESCRIPTION
**Pull request by Jamiras**

### CODE REVIEW

File: app/Helpers/database/player-achievement.php [hunk 1]
```diff
@@ -40,74 +40,64 @@ function playerHasUnlock(?string $user, int $achievementId): array
     return $retVal;
 }
 
-function unlockAchievement(string $user, int $achIDToAward, bool $isHardcore): array
+function unlockAchievement(string $username, int $achIDToAward, bool $isHardcore): array
```
Comment (luchaos) [inline app/Helpers/database/player-achievement.php:43]: `int $achievementId` for simpler parameter naming?

**Review by luchaos — APPROVED** [review]
Awesome 👏

File: app/Helpers/database/player-achievement.php (hunk 1 above)
Comment (Jamiras) [reply app/Helpers/database/player-achievement.php:43]: I plan to follow this up with a migration to a command. I'll make the change there.

Comment (Jamiras) [reply app/Helpers/database/player-achievement.php:43]: Looks like there's already a partially implemented `UnlockPlayerAchievementAction` using the new schemas. I'll make the change here.

**Pull request merged**